pandas/tests/generic/test_generic.py

```diff
@@ -583,6 +583,15 @@ def test_pct_change(self, periods, fill_method, limit, exp):
 class TestNDFrame(object):
     # tests that don't fit elsewhere
 
+    def test_to_excel_size(self):
+        BREAKING_SHAPE = (2**20 + 1, 2**14 + 1)
+        arr = np.zeros(shape=BREAKING_SHAPE)
+        df = pd.DataFrame(arr)
+        filepath = 'test.xlsx'
+
+        with pytest.raises(ValueError):
+            df.to_excel(filepath)
+
```

Comment:
* Move to Excel tests
* Reference issue number as a comment beneath the test function `def`.

Reply:
will do.

Reply:
I'm trying to move the new test into pandas/tests/io/test_excel.py. Before making any changes I ran the test suite on a recently checked out version of code from master and 16 tests fail. Is that expected behavior?

Reply:
It looks like of the failures are generated because of your test.  Trying to allocate space for that large of an `ndarray` is causing a `MemoryError`.